production/terraform/modules/s3/variables.tf

```diff
@@ -0,0 +1,28 @@
+variable "region" {
+  description = "AWS region"
+  type        = string
+  default     = "us-east-2"
+}
+
+variable "bucket_name" {
+  description = "Bucket name for Loki storage"
+  type    = string
+  default = "loki-data"
```

Comment:
as long as the s3 bucket name must be unique, I think we need to remove the default value here and add validation that it's not empty to check the user actually defined it .... wdyt?

Reply:
That is a very good point. I've forgot that bucket names must be unique across all accounts.